xmodule/xml_block.py

```diff
@@ -324,8 +329,8 @@ def parse_xml(cls, node, runtime, _keys, id_generator):
         # Note: removes metadata.
         definition, children = cls.load_definition(definition_xml, runtime, def_id, id_generator)
 
-        # VS[compat] -- make Ike's github preview links work in both old and
-        # new file layouts
+        # VS[compat]
+        # Make Ike's github preview links work in both old and new file layouts.
```

Comment:
What does it mean? Do we still need this?

Reply:
@Agrendalath, I couldn't figure this out for sure. This was added in https://github.com/openedx/edx-platform/commit/932a9be7479c38701d84eb0a070e7c17dde528c6, and perhaps is needed for [add_staff_markup](https://github.com/openedx/edx-platform/blob/20fac60c0c5b738cd2ef817fea93805159c83f21/openedx/core/lib/xblock_utils/__init__.py#L263-L352) function, but I don't have any context on https://github.com/MITx to confidently say whether this is still needed. My wild guess: some courses were stored on GitHub, and this was needed to generate "edit" GitHub links for staff.